book/src/proposals/tower-lockout-slashing.md

```diff
@@ -10,9 +10,10 @@ slashed by some percentage.
 ## Votes
 
 Validators submit votes which contain a list of slots.  To be
-accepted, the vote must contain the list of slots currently in the
-VoteState program, along with new votes.  The slots must match the
-VoteState exactly.
+accepted, the vote must contain the list of N slots currently in
+the VoteState program, along with new votes.  The first `N - new
+votes`, must match the VoteState exactly.  N should be large enough
```

Comment:
Hmm, do you men `2^(sizeof(N))` should be longer than a full epoch? How big can `new votes` be? I think it would be good to give an example of such a vote with these two fields, the `N` and the `new votes`, and why it's important that the N votes `enforce a lockout that spans a full epoch.`